Sources/docc/DocCDocumentation.docc/Commands/index.md

```diff
@@ -0,0 +1,32 @@
+# ``index``
+
+Create an LMDB index for an already built documentation archive.
+
+@Metadata {
+    @PageImage(purpose: icon, source: command-icon)
+}
+
+If you pass the `--emit-lmdb-index` flag to the ``convert`` command, `docc` will create an LMDB index during the build so that you don't need to process the archive after it's been built.
```

Comment:
Maybe add another sentence here explaining that this is equivalent to using `docc convert --emit-lmdb-index`, and that you would use this when the archive was already created.